dash/testing/application_runners.py

```diff
@@ -179,23 +179,34 @@ def __init__(self, keep_open=False, stop_timeout=3):
         self.proc = None
 
     # pylint: disable=arguments-differ
-    def start(self, app_module, application_name="app", port=8050):
+    def start(
```

Comment:
@alexcjohnson  played a little bit on the dash-docs side

this is somehow a little break change, but I guess it's not used by anyone in the community.  the raw_command option would give us more flexibility in the future.  like in dash-docs case, our usage should just use python run.py instead a gunicorn or waitress which targets for production.